src/refract/RenderJSONVisitor.cc

```diff
@@ -102,10 +105,38 @@ namespace refract
         if (!a || a->value.empty()) {
             return NULL;
         }
+        
+        std::cerr << "samples" << std::endl;
 
         return TypeQueryVisitor::as<T>(*(a->value.begin()));
     }
 
+    template<typename T>
+    const T* getNullable(const T& e) {
+        IElement::MemberElementCollection::const_iterator ta = e.attributes.find("typeAttributes");
+        
+        if (ta == e.attributes.end()) {
+            return NULL;
+        }
+        
```

Comment:
@pksunkara I'm still pretty stuck here, I have a list of notes in gitter but the short version is that I believe when `getNullable` hits the string element representing our nullable element, it does not contain any attributes whatsoever. I came to this conclusion by adding a breakpoint and reading the value of e every time `getNullable` is called. I have some screenshots of the data in gitter that should help show what I found.

Do you think we can still find the attributes for string elements or will we need to update the serialization of string elements?